src/styles/Book.css

```diff
@@ -0,0 +1,288 @@
+    transform: rotate(180deg);
+  }
+
+  100% {
+    z-index: 100;
+    transform: rotate(315deg);
+  }
+}
+
+.progress-percent {
+  width: 4.25rem;
+  height: 2.438rem;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progress-completed {
+  width: 5rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.progress-percent,
+.category,
+.progress-completed {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.list-block,
+.right-section {
+  justify-content: space-between;
+}
+
+.list-actions {
+  gap: 30px;
+  margin-top: 20px;
+}
+
+.list-actions button {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.list-actions span {
+  color: #4386bf;
+}
+
+.right-section {
+  gap: 70px;
+  align-items: center;
+}
+
+.list-block {
+  list-style-type: none;
+}
+
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title,
+.book-author,
+.list-actions button,
+.current-chapter,
+.chapter-number,
+.update-progress {
+  font-family: 'Roboto Slab', serif;
+}
+
+.book-title {
+  width: 15.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-author {
+  width: 8.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.div {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 0.1rem 0 3.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.chapter {
+  margin-left: 0;
+}
+
+.current-chapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter-number {
+  width: 5.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.update-progress {
```

Comment:
- [ ] [Optional] Well! Paying close attention to design details is crucial for achieving a polished look. Currently, the `.update-progress` button has a font-weight of 400, but according to the design, it should be 300. To ensure your project aligns with the design as closely as possible, please update your code to set the font weight to 300 for this button.

By making this adjustment, you'll enhance the consistency of your project's appearance with the design guidelines. Keep up the great work! 👍🎨💼